pkg/utils/etcdutil/etcdutil.go

```diff
@@ -865,6 +865,16 @@ func (lw *LoopWatcher) load(ctx context.Context) (nextRevision int64, err error)
 	if limit != 0 {
 		limit++
 	}
+	if err := lw.preEventsFn([]*clientv3.Event{}); err != nil {
+		log.Error("run pre event failed in watch loop", zap.String("name", lw.name),
+			zap.String("key", lw.key), zap.Error(err))
+	}
+	defer func() {
```

Comment:
Will it affect the keyspace related logic?

Reply:
I will check it

Reply:
This PR will also call `postEventsFn` when meeting failures in loading data.

After my check,  keyspace-watcher will call `postEventsFn` to retry, keyspace-server-watcher will call `postEventsFn` to send keyspaces.

So I modify a little about it.

Reply:
The return error also changes, right?